zkevm-circuits/src/tx_circuit.rs

```diff
@@ -559,7 +566,8 @@ impl<F: Field> SubCircuitConfig<F> for TxCircuitConfig<F> {
             cb.gate(meta.query_fixed(q_enable, Rotation::cur()))
         });
 
-        // TODO: add constraints for AccessListGasCost.
+        // TODO: add constraints for AccessListAddressesLen, AccessListStorageKeysLen,
+        // AccessListGasCost and AccessListRLC.
```

Comment:
```suggestion
        // TODO: add constraints for AccessListAddressesLen, AccessListStorageKeysLen
        // and AccessListRLC.
```

Reply:
Update this comment in the same commit [6880378](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/68803783b1447b8fefe2c1fabd626beae51b8a83).